modules/src/mock/context.rs

```diff
@@ -124,19 +128,60 @@ pub struct MockContext {
 
     /// ICS26 router impl
     router: MockRouter,
+
+    /// Object capabilites impl
+    ocap: Arc<Mutex<MockOCap>>,
 }
 
 /// Returns a MockContext with bare minimum initialization: no clients, no connections and no channels are
 /// present, and the chain has Height(5). This should be used sparingly, mostly for testing the
 /// creation of new domain objects.
 impl Default for MockContext {
     fn default() -> Self {
+        let ocap = Arc::new(Mutex::new(MockOCap::default()));
         Self::new(
             ChainId::new("mockgaia".to_string(), 0),
             HostType::Mock,
             5,
             Height::new(0, 5),
         )
+        .with_router(dummy_router(ocap.clone()))
+        .with_ocap(ocap)
+        .with_port_capability(PortId::default(), dummy_module_id())
+    }
+}
+
+impl Clone for MockContext {
```

Comment:
We manually implement `Clone` because `ocap` is a pointer. This implementation sets up the ocap pointer to it's initial state as per the `Default` impl.